addons/website_hr_recruitment/controllers/main.py

```diff
@@ -116,6 +116,18 @@ def jobs_thankyou(self, **post):
         # Retro-compatibility for saas-3. "phone" field should be replace by "partner_phone" in the template in trunk.
         value['partner_phone'] = post.pop('phone', False)
 
+        is_partner_list = request.session.model('res.partner').search([('email', '=', post.get('email_from').replace(" ", ""))], 0, 0, 0, request.session.context)
+        if not is_partner_list:
+            value_partner = {
+                'name': post.get('partner_name'),
+                'email': post.get('email_from'),
+                'phone': value['partner_phone'],
+            }
+            partner = env['res.partner'].create(value_partner)
```

Comment:
This will crash if you are not logged in (public user by example). I think you should use "sudo()"